lib/elixir/lib/string.ex

```diff
@@ -1900,10 +1900,9 @@ defmodule String do
   @doc """
   Returns a float whose text representation is `string`.
 
-  `string` must be the string representation of a float.
-  If a string representation of an integer wants to be used,
-  then `Float.parse/1` should be used instead,
-  otherwise an argument error will be raised.
+  `string` must be the string representation of a float including a decimal point.
+  In order to parse a string without decimal point as a float then `Float.parse/1`
+  should be used. Otherwise, an argument error will be raised.
```

Comment:
Nitpick, but can we say "Otherwise, an `ArgumentError` exception will be raised"? :)